symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/stream/OboStreamService.java

```diff
@@ -0,0 +1,37 @@
+package com.symphony.bdk.core.service.stream;
+
+import com.symphony.bdk.core.api.invoker.ApiException;
+import com.symphony.bdk.core.api.invoker.ApiRuntimeException;
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.gen.api.StreamsApi;
+import com.symphony.bdk.gen.api.model.StreamAttributes;
+import com.symphony.bdk.gen.api.model.StreamFilter;
+import com.symphony.bdk.gen.api.model.V2StreamAttributes;
+
+import java.util.List;
+
+class OboStreamService {
+
+  protected final StreamsApi streamsApi;
+
+  protected OboStreamService(StreamsApi streamsApi) {
+    this.streamsApi = streamsApi;
+  }
+
+  public V2StreamAttributes getStreamInfo(String streamId, AuthSession oboSession) {
```

Comment:
I don't know why, but I'd prefer to have `AuthSession` as first parameter. Same for next method